test/mock.js

```diff
@@ -239,29 +239,39 @@ const getEventMetadataResponse = {
   event_code: 'com.adobe.event_code_1'
 }
 
-const createEventMetadataBadRequest =
-{
+const createEventMetadataBadRequest = {
   label: 'test event code 1',
   description: 'Test for SDK 1'
 }
 
-const createWebhookRegistration =
-{
+const createRegistration = {
   name: 'name',
   description: 'description',
   client_id: 'test-apikey',
   webhook_url: 'https://test-webhook',
-  delivery_type: 'WEBHOOK',
+  delivery_type: 'webhook',
```

Comment:
Is lowercase required now? What happens if someone specifies this in uppercase?

Reply:
nothing, it will still be valid. we ignore case. Just putting it here because that is what is in the response as well.